src/Aspire.Hosting/ApplicationModel/EndpointReference.cs

```diff
@@ -153,9 +153,9 @@ public class EndpointReferenceExpression(EndpointReference endpointReference, En
     /// <summary>
     /// Gets the value of the property of the endpoint.
     /// </summary>
-    /// <param name="cancellationToken"></param>
-    /// <returns></returns>
-    /// <exception cref="InvalidOperationException"></exception>
+    /// <param name="cancellationToken">A <see cref="CancellationToken"/>.</param>
```

Comment:
Some places say "a cancellation token", others say "the cancellation token". Should pick one and be consistent.